api/sse/sse_service.py

```diff
@@ -0,0 +1,49 @@
+from functools import wraps
+from typing import List
+
+from django.conf import settings
+
+from . import tasks
+
+
+def _sse_enabled(get_project_from_first_arg=lambda obj: obj.project):
```

Comment:
I think it would be good to have some documentation on this decorator or even on this module explaining what's happening here?

Reply:
I have added one, let me know if it makes sense